docs/NativeComponentsIOS.md

```diff
@@ -43,6 +43,7 @@ RCT_EXPORT_MODULE()
 
 @end
 ```
+**Note:** Do not attempt to set the `frame` or `backgroundColor` properties on the `UIView` instance that you vend through the `-view` method. The values set by your custom class will be overwritten by the React Native layout/rendering process. If you need this grandular control it might be better to compose or wrap the `UIView` instance you want to style in another `UIView` and return the wrapper `UIView` instead. See [Issue 2948](https://github.com/facebook/react-native/issues/2948) for more context.
```

Comment:
> The values set by your custom class will be overwritten by the React Native layout/rendering process.

Can you make this more precise, like "React Native will overwrite the values set by your custom class in order to match your JS component's layout props."

> If you need this grandular control

"this granularity of control"